applications/compare_cumulative_psf.py

```diff
@@ -35,14 +35,13 @@ def getData(**kwargs):
 
 if __name__ == '__main__':
     sourceDistance = 12 * u.km
-    site = 'south'
     version = 'prod4'
     label = 'lst_integral'
     zenithAngle = 20 * u.deg
     offAxisAngle = [0 * u.deg]
 
     tel = TelescopeModel(
-        telescopeName='north-lst-1',
+        telescopeType='north-lst-1',
```

Comment:
Why did you change it to telescopeType? It shouldn't work now. I changed it to telescopeName every where.

Reply:
A mistake that was propagated accidentally because I had to merge in order to test the Docker.
Sorry about that.

Reply:
No problem. I think we should fix it directly in the master, just to prevent propagating it. Can you do that or I do?

Reply:
I don't think I am allowed to push directly to the master, at least it was rejected last time I tried...

Reply:
That is weird. I'm allowed.
I'll fix it now and check what is going on in your case.

Reply:
I learned how to set this kind of restriction but there is none set for this repository. Maybe you could try again someday jut to be sure.

Reply:
OK, weird, maybe I misunderstood a previous error. Will try again in the future.